differs/pipDiff.go

```diff
@@ -37,27 +39,50 @@ func getPythonVersion(pathToLayer string) ([]string, error) {
 	return matches, nil
 }
 
-func (d PipDiffer) getPackages(path string) (map[string]map[string]utils.PackageInfo, error) {
-	packages := make(map[string]map[string]utils.PackageInfo)
+func getPythonPaths(vars []string) []string {
```

Comment:
Good point. I believe this would be the PYTHONPATH set at the time the container is exported into an image, so yes, if it's been overwritten and not merely added to while the container is running, you wouldn't have all the old ones... @nkubala thoughts on this?